ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -61,18 +63,18 @@ RUN echo "@today_str"
 USER buildfarm
 ENTRYPOINT ["sh", "-c"]
 @{
+cmd = 'PATH=/usr/lib/ccache:$PATH' + \
+      ' PYTHONPATH=/tmp/ros_buildfarm:$PYTHONPATH python3 -u'
```

Comment:
The indentation is still off. The quotes should not align but the line should simply be indented one level aka four spaces. Same below with the cmd concatenation.